internal/cluster/cluster.go

```diff
@@ -222,6 +222,8 @@ type ClusterSpec struct {
 	InitTimeout *Duration `json:"initTimeout,omitempty"`
 	// Interval to wait for a db to be synced with a master
 	SyncTimeout *Duration `json:"syncTimeout,omitempty"`
+	// Interval to wait for a db to boot and become ready
+	DBWaitReadyTimeout *Duration `json:"dbWaitReadyTimeout,omitempty"`
```

Comment:
s.DBWaitReasyTimeout is not used by the keeper that will continue to use DefaultDBWaitReadyTimeout. You should also change cmd/keeper/cmd/keeper.go

Reply:
🤦‍♂ sorry about that! Cherry picked one commit and missed the other! Updated.